briskedu/brisk-learning
# Make chrome extension id stable

Summary:
This makes sure our extension id is stable so that when it's install in new places it doesn't change.
https://app.asana.com/0/1203963408726580/1204099843417544
Test Plan:
1. Remove and add extension on two machines and confirm id is the same
2. Modify code and remove and remove extension and confirm id is still the same
3. Update .env in api and confirm extension still works
